app/com/linkedin/drelephant/tuning/AutoTuningOptimizeManager.java

```diff
@@ -0,0 +1,44 @@
+package com.linkedin.drelephant.tuning;
```

Comment:
Can you add Dr. Elephant's license header?

Reply:
done